src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -74,6 +77,11 @@ internal FakeManager(Type fakeObjectType, object proxy)
         /// </summary>
         public virtual Type FakeObjectType { get; }
 
+        /// <summary>
+        /// Gets the name of the fake.
+        /// </summary>
+        public string FakeObjectName { get; }

```

Comment:
This isn't really necessary, but I see no harm in it, and some people might find it useful.